packages/space-opera/src/components/shared/expandable_content/expandable_section.css.ts

```diff
@@ -25,16 +25,12 @@ export const styles: CSSResult = css`
   max-height: 0;
   overflow: hidden;
   padding: 0 20px;
-  transition-duration: 300ms;
```

Comment:
no more transitions?

Reply:
I don't think they improve the user experience, especially if this is a situation where users may be opening and closing many sections